GOVERNANCE.md

```diff
@@ -18,129 +26,133 @@ the cert-manager GitHub org must follow the guidelines in [the contributing
-    - Sponsors must be maintainer.
-- Open a PR to update the OWNERS file
-    - Ensure your sponsor is @mentioned on the PR.
-    - Include a list of the 5 PRs you reviewed & /lgtm'ed.
+### Becoming an Approver
 
-### Privileges
-- Can /approve on pull requests
+To become an approver and start merging PRs, you must have reviewed 5 PRs.
+
+You will then need to get sponsorship from one of the maintainers. The
+maintainer sponsoring you must have had close work interactions with you and be
+knowledgeable of some of your work. 
+
+To apply, open a PR to update the `OWNERS` file and mention your sponsor in the
+description. The PR description should also list the PRs you have reviewed.
 
 ### Responsibilities
+
 - Expected to be responsive to review requests.
-- Stay up to date with the project's direction and goals. eg. by attending the weekly and/or bi-weekly meetings.
+- Stay up to date with the project's direction and goals,
+  e.g., by attending some of the bi-weekly meetings, standups,
+  or being around in the cert-manager-dev Slack channel.
+
+### Privileges
 
-## Maintainer Role
+- Can `/approve` on pull requests.
 
-= (in CNCF terms) maintainer
+## Maintainer
 
-Someone who can communicate with the CNCF on behalf of the project and who can
-participate in a "maintainers vote".
+A maintainer is someone who can communicate with the CNCF on behalf of the
+project and who can participate in a maintainers vote. The list of maintainers
+is available in the file [`MAINTAINERS.md`](./MAINTAINERS.md).
 
-**Defined by:** the list in the [MAINTAINERS.md](./MAINTAINERS.md) file
+### Becoming a Maintainer
 
-### Requirements
-- You must be an approver.
-- You have successfully reviewed & /approved'ed 10 PRs.
-- Must be able to dedicate time to participate in maintainer meetings.
-- Must be able to dedicate time to participate in maintainer votes.
-- Must be able to dedicate time to monitor the cert-manager-\* mailing lists and help out when possible.
-- Must be able to dedicate time to rapidly respond to any time-sensitive security release processes.
-- Must be able to dedicate time to attend meetings with the cert-manager Steering Committee.
-- Must be able to dedicate time to communicate with the CNCF on behalf of the project.
+Anyone can become a cert-manager maintainer. Maintainers should be proficient in
+Go; have expertise in at least one of the domains (Kubernetes, PKI, ACME); have
+the time and ability to meet the maintainer expectations above; and demonstrate
+the ability to work with the existing maintainers and project processes.
+
+To become a maintainer, start by expressing interest to existing maintainers.
+Existing maintainers will then ask you to demonstrate the qualifications above
+by contributing PRs, doing code reviews, and other such tasks under their
+guidance. After several months of working together, maintainers will decide
+whether to grant maintainer status.
```

Comment:
I don't really agree with this section, I think this section contains tasks for approvers and reviewers.

Reply:
Good point, I wrongly copy paste the old Becoming a Maintainer. I’ll fix it